.github/workflows/acceptance_tests_common.yml

```diff
@@ -84,4 +87,13 @@ jobs:
         run: ./testsuite/podman_runner/18_split_secondary_p_tests.sh
       - name: run_secondary_tests
         run: ./testsuite/podman_runner/${{ inputs.secondary_tests }} 
+      - name: get_server_logs
+        if: ${{ failure() }}
```

Comment:
If it doesn't cost resources... I would save the logs always, not only when failing.

Reply:
It takes time ... 5 minutes aprox. There is a limit on disc space too, but I do not think  we have a problem there. I rather prefer to not take these extra 5 minutes than to have the logs when it succeeds.

Reply:
What do you think?

Reply:
Let's skip it, and if we ever find a good reason to have the logs in passed tests, we can think about adding them in some conditional way (using some label in the GitHub PR).

Reply:
We agreed on slack that it is ok to skip.